README.md

```diff
@@ -636,7 +636,7 @@ Command                                                     | Description
 ## Environment variables
 
 
-- `RUSTUP_HOME` (default: `~/.rustup` or `%USERPROFILE%/.rustup`)
+- `RUSTUP_HOME` (default: `~/.rustup` or `%USERPROFILE%\.rustup`)
```

Comment:
This is a markdown file, does the backslash need escaping here in order to render properly, or do the backticks sort that out?

Reply:
No, this renders properly as is. The change was more for consistency with Windows users. Backslash to me indicates this is the windows path, versus a forward slash. I can remove this if you'd like.

Reply:
Oh no, I agree with the change from a Windows user expectation perspective, that's great, thanks for the confirmation.